src/industry.h

```diff
@@ -33,6 +33,13 @@ enum ProductionLevels {
 	PRODLEVEL_MAXIMUM = 0x80,  ///< the industry is running at full speed
 };
 
+enum class IndustryAction : byte {
+	SetControlFlags = 0,       ///< Set IndustryControlFlags
+	SetExclusiveSupplier = 1,  ///< Set exclusive supplier
+	SetExclusiveConsumer = 2,  ///< Set exclusive consumer
+	SetText = 3,               ///< Set additional text
```

Comment:
Coding style for enum members:
> LordAro: obviously it doesn't need the prefix, but yeah, should probably still be all caps

Reply:
Other 2 enum classes (ElementFloat and MidiSysexMessage) use this syntax. And IMO it's much nicer, as they are not in global namespace so THERE IS NO NEED TO SHOUT :p

Reply:
Well, letting @LordAro have the last word here, but otherwise 👍